.github/workflows/ci.yml

```diff
@@ -26,7 +26,10 @@ jobs:
           use-mamba: true
           python-version: ${{ matrix.PYTHON_VERSION }}
           environment-file: environment.yml
-      - shell: bash -x -l {0}
+      - name: Run the unittests
+        shell: bash -x -l {0}
         run: |
+          mamba install google-cloud-storage
```

Comment:
Can you add that to the `environment.yaml`?

Reply:
We're current inconsistent about which client libraries are part of the `environment.yaml`. Azure isn't, GC isn't, `dulwich` (git-store) is. I like keeping them as optional dependencies, but you should we should add them?

Reply:
We should add them all. Currently, Azure and GCS are omitted because we don't have running tests for them on CI. We should add them in the next days all though.